service/src/main/kotlin/fi/espoo/evaka/messaging/message/MessageQueries.kt

```diff
@@ -29,6 +29,25 @@ fun Database.Read.getUnreadMessagesCount(
         .one()
 }
 
+fun Database.Transaction.markThreadRead(accountId: UUID, threadId: UUID): Int {
+    // language=SQL
+    val sql = """
+UPDATE message_recipients rec
+SET read_at = :readAt
+FROM message msg
+WHERE rec.message_id = msg.id
+  AND msg.thread_id = :threadId
+  AND rec.recipient_id = :accountId
+  AND read_at IS NULL;
+    """.trimIndent()
+
+    return this.createUpdate(sql)
+        .bind("accountId", accountId)
+        .bind("threadId", threadId)
+        .bind("readAt", HelsinkiDateTime.now())
```

Comment:
Why `HelsinkiDateTime.now()` instead of `now()` in the SQL query?

Reply:
🤷